packages/api-specification/interface.ts

```diff
@@ -583,6 +583,23 @@ declare namespace ssf {
      */
     body?: string;
   }
+
+  interface NotificationPermissionCallback {
+    (permission: String): void;
+  }
+
+  class Notification {
+    /**
+     * Create a notification
+     * @param title - The title text of the notification.
+     * @param options - The notification options.
+     */
+    constructor(title: string, options: NotificationOptions);
+    
+    static permission: string;
```

Comment:
I think we could add typedoc comments here so we get descriptions for the documentation. Might be good to have `static permission: 'granted' | 'denied' | 'default'` so we can see the possible values.